plugins/UCRM-Client-Signup/README.md

```diff
@@ -0,0 +1,34 @@
+# UCRM Client Signup
+A [UCRM plugin](https://github.com/Ubiquiti-App/UCRM-plugins) that provides a frontend form for clients to signup.
+
+_Developed by [Charuwts, LLC](https://charuwts.com)_
+
+When installed onto UCRM the plugin public URL will display a form that anyone can enter valid info into to create a client.
+
+## Instructions
+
+_*Not ready for Production.*_
```

Comment:
why not?

Reply:
Technically it is now, but I haven't written many tests so I personally wouldn't consider it secure/stable. But if someone wanted to use it live I wouldn't have a problem with it. Would you suggest I remove the comment?

Reply:
I will get back to you about it. :)

Reply:
IMO it should stay, but maybe more verbose with something like you just commented here.

Reply:
Other than that, I think this can be merged. @UBNT-Petr

Reply:
Thank you @keksa

Reply:
FYI, I added clarification to the readme. @keksa @UBNT-Petr